rholang/main/runner.py

```diff
@@ -3,11 +3,15 @@
 
 from contextlib import contextmanager
 from io import StringIO
+from os.path import relpath
```

Comment:
`relpath` is ambient authority: its results are different depending on platform

Reply:
Interesting. Do you have some further reading on this? I've always just assumed if I used os.path related calls, I would be fine.

Reply:
Also any suggestion on how to do the equivalent of relpath in a ocaap disciplined fashion? If it is quicker for you to just submit a commit on top and have me see, then please feel free to do so.

Reply:
I'm going to merge this as is so we can move to the `/rchain` repo and then we can pick this back up.